book/src/development/adding_lints.md

```diff
@@ -122,17 +125,18 @@ fn main() {
 }
 ```
 
-Now we can run the test with `TESTNAME=foo_functions cargo uibless`, currently
-this test is meaningless though.
+Note that we are adding comment annotations with the name of our lint to mark
+lines where we expect an error. Once we have implemented our lint we can run
+`TESTNAME=foo_functions cargo uibless` to generate the `.stderr` file.
```

Comment:
Should we talk about the `.fixed` files too? Those will be generated and must be checked in as well.

Reply:
Fixed in 5d7e55c